spec/factories/users.rb

```diff
@@ -3,5 +3,6 @@
     email 'doorsat8@mailinator.com'
```

Comment:
I would think about moving the email and username to be sequencing at some point. Doesn't need to be in this PR